### DISCUSSION TITLE
Leaking real IPv6 address

### DISCUSSION THREAD

**Discussion by iopq** [v2rayA/v2rayA#1491] [General]
While my IPv4 address shows up as the proxy one, on ipleak.net I see my ipv6 which should never be leaked

furthermore, yahoo mail blocks me from going there upon seeing my real ipv6 so I have to completely disable it

in v2rayng on android I don't leak my ipv6, I'm seeing the proxy's ipv6

**Comment by Smart-SangGe** [comment]
same problem

**Comment by salehMomtaz** [comment]
Same problem, i will share details later, odds are that i have this problem not only pc but also android

**Comment by linusboyle** [comment]
I have the same problem.
